src/Uno.Toolkit.UI/Helpers/MediaGallery/MediaGallery.Android.cs

```diff
@@ -0,0 +1,128 @@
+﻿#if __ANDROID__
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Provider;
+using Android.Webkit;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel;
+using Windows.Extensions;
+using Path = System.IO.Path;
+using Stream = System.IO.Stream;
+using Environment = Android.OS.Environment;
+using NativeFile = Java.IO.File;
+using NativeUri = Android.Net.Uri;
+using static Android.Provider.MediaStore;
+
+namespace Uno.Toolkit.UI;
+
+partial class MediaGallery
+{
+	private static readonly DateTime _unixStartDate = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+	private static async Task<bool> CheckAccessPlatformAsync()
+	{
+		if ((int)Build.VERSION.SdkInt < 29)
+		{
+			return await PermissionsHelper.CheckWriteExternalStoragePermission(default);
+		}
+		else
+		{
+			return true;
+		}
```

Comment:
I know the APIs for storage access changed but we actually don't need any specific permissions for external file storage anymore? Seems odd

Reply:
It is quite interesting but not:

![image](https://github.com/unoplatform/uno.toolkit.ui/assets/1075116/b5d6c44f-4981-4a84-9846-cc10dc8d640b)